packages/components/src/AuthenticatedRoute.tsx

```diff
@@ -26,21 +25,26 @@ const userQuery = gql`
 export const AuthenticatedRoute = ({
   render,
   redirectTo,
+  signupUrl,
   onlyAllowUnauthenticated = false,
-  allowWithoutEth = false,
-  ethSignupPath,
   ...otherProps
 }: AuthenticatedRouteProps) => {
   const auth = getApolloSession();
 
   const hasAuthToken = !!auth && !!auth.token;
 
-  if (onlyAllowUnauthenticated === hasAuthToken) {
-    return <Redirect to={redirectTo} />;
+  if (onlyAllowUnauthenticated) {
```

Comment:
this boolean logic is kinda weird, why not:

if (onlyAllowUnauthenticated && hasAuthToken)
else if (!hasAuthToken)

Reply:
Noted, I'm going to leave this as is for this PR, but I'll take care of it in a PR for tomorrow.